GenumKit/Parsers/StringsFileParser.swift

```diff
@@ -17,12 +31,41 @@ public final class StringsFileParser {
 
   // Localizable.strings files are generally UTF16, not UTF8!
   public func parseStringsFile(path: String) throws {
+    guard let data = NSData(contentsOfFile: path) else {
+      throw StringsFileParserError.FailureOnLoading(path: path)
+    }
+
+    let plist = try NSPropertyListSerialization.propertyListWithData(data, options: NSPropertyListReadOptions.Immutable, format: nil)
```

Comment:
The reason why I didn't use `Dictionary(contentsOfFile: )` is that when it fails, it just returns `nil`. `Dictionary(contentsOfFile: )` can fail with various reasons: a file does not exist, an invalid encoding, malformed plist file, etc.. But we are not able to know the reason of the failure with `Dictionary(contentsOfFile:)`.

`NSPropertyListSerialization.propertyListWithData(..)` throws an error by itself. By using it, we can propagate errors with rich information that should be helpful for users.

Anyway, I also agree that `let plist = Dictionary(contentsOfFile: path)` is much more simple and easy to read. It seems that both have pros and cons so I will follow your decision.